src/pages/therapists/index.tsx

```diff
@@ -1,34 +1,128 @@
-import { Container, InputBase, IconButton } from '@mui/material';
-import Box from '@mui/material/Box';
+import { useState, useEffect, ChangeEvent } from 'react';
+import {
+  Container, InputBase, IconButton, Box, Grid, Pagination,
+} from '@mui/material';
 import SearchIcon from '@mui/icons-material/Search';
-import Grid from '@mui/material/Grid';
-import GridCard from '../../components/TherapistCard';
+import { enqueueSnackbar } from 'notistack';
+import { AxiosError } from 'axios';
+import { SearchBoxStyle } from './classes';
+import { TherapistList } from '../../components';
+import { axiosInstance } from '../../utils/apis';
 
-const Therapist = () => (
-  <>
-    <Container>Therapist Page</Container>
-    <Box sx={{ flexGrow: 1 }}>
+const TherapistPage = () => {
+  const [therapists, setTherapists] = useState([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(
+    () => {
+      const fetchData = async () => {
+        try {
```

Comment:
try to rename the function like getTherapistData

Reply:
Done